public/app/features/dashboard-scene/scene/DashboardSceneRenderer.tsx

```diff
@@ -26,6 +27,20 @@ export function DashboardSceneRenderer({ model }: SceneComponentProps<DashboardS
   const navModel = getNavModel(navIndex, 'dashboards/browse');
   const hasControls = controls?.hasControls();
 
+  // Remember scroll pos when going into view panel, edit panel or settings
+  useMemo(() => {
+    if (viewPanelScene || editview || editPanel) {
```

Comment:
Here I found an edge case with `editview` (Settings), if we go to settings, and click on the variables tab(or other tab), the scroll position is updated to zero. 

https://github.com/user-attachments/assets/020bbcd2-7535-40dc-8580-94214f5bb257

Reply:
@axelavargas good catch, easy to fix